plugins/by-name/blink-cmp/default.nix

```
@@ -42,4 +42,18 @@ lib.nixvim.plugins.mkNeovimPlugin {
     };
     signature.enabled = true;
   };
+
+  extraOptions = {
+    setupLspCapabilities = lib.nixvim.options.mkEnabledOption "setup LSP capabilities for blink-cmp";
+  };
+
+  extraConfig = cfg: {
+    plugins.lsp.capabilities =
+      lib.mkIf cfg.setupLspCapabilities # lua
+        ''
+          -- Capabilities configuration for blink-cmp
+          capabilities.textDocument.completion.completionItem.snippetSupport = true
+          capabilities = require("blink-cmp").get_lsp_capabilities(capabilities)
```

```suggestion
          capabilities = vim.tbl_deep_extend('force', capabilities, require('blink.cmp').get_lsp_capabilities())
```

We don't want to replace all capabilities. I have this in my config to make sure we extend the capabilities.